--- asymptote-2.61/Makefile.in.orig	2019-11-18 06:27:29.000000000 +0100
+++ asymptote-2.61/Makefile.in	2019-12-23 20:22:06.892642859 +0100
@@ -137,7 +137,6 @@
 	echo @set VERSION $(revision) > doc/version.texi
 	echo @set Datadir @datadir@ >> doc/version.texi
 
-	echo "#!/usr/bin/env python3" > GUI/xasyVersion.py
 	echo xasyVersion = \"$(revision)\" >> GUI/xasyVersion.py
 
 	if test -n "$(MSDOS)"; then \
